moztrap/model/core/management/commands/cleanup_test_data.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from datetime import datetime
+from optparse import make_option
+
+from django.core.management.base import BaseCommand
+
+from moztrap.model.core import models as core_models
+from moztrap.model.environments import models as env_models
+
+
+class Command(BaseCommand):
+    help = 'Deletes old test data'
+    option_list = BaseCommand.option_list + (
+        make_option('--permanent',
+                    action='store_true',
+                    dest='permanent',
+                    default=True,
+                    help='Permanently delete records?'),)
+
+    def handle(self, *args, **options):
+        for model in (core_models.Product,
+                      env_models.Category,
+                      env_models.Element,
+                      env_models.Profile):
```

Comment:
I think we need ``env_models.Environment`` in this list as well.  The order may matter, but I'm not certain.  Might need to be ``Profile``, ``Environment``, ``Element``, ``Category``.  But you should be sure to test it on your local machine, of course.  :)